frontend/src/routes/onboard/onboard.tsx

```diff
@@ -302,16 +303,51 @@ export function OnboardWalletImport() {
 }
 
 export function OnboardNetwork() {
+  const { state, actions, dispatch } = useGlobal()
+  // The current view of the drawer
+  const [view, setView] = React.useState<'add' | 'edit' | 'manage'>('manage')
```

Comment:
suggestion(non-blocking): 'add' | 'edit' | 'manage' is a repetition similar to above. Perhaps best to redeclare these as typescript enums

Reply:
i'd rather skip this, as the onboarding is getting removed in the next PR related to https://github.com/vegaprotocol/vegawallet-desktop/issues/269. this is a temporary solution to leave things in a working state meanwhile